src/service.js

```diff
@@ -9,14 +9,15 @@ class UploaderService {
     upload({formData = new FormData(), field, additionalFields, customHeaders}) {
         return new Promise((resolve, reject) => {
 
-            formData.append(field, this.file);
-
             if (additionalFields) {
                 for (const fieldName in additionalFields) {
                     formData.append(fieldName, additionalFields[fieldName]);
                 }
             }
 
+            formData.append('content-type', this.file.type);
```

Comment:
not that this is not 100% foolproof. It would work only for common filetypes, and only by extension (and not file headers).

Are we cool with that?

Reply:
Yeah, it will be better to use something like this:https://medium.com/the-everyday-developer/detect-file-mime-type-using-magic-numbers-and-javascript-16bc513d4e1e

I added `file.type` as a first step